data/changelog.txt

```diff
@@ -43,6 +43,7 @@ Artifacts regenerate ammo option.
 Added ACTIVE enchantment condition.
 Allow recipes to specify individual activity levels.
 Add weariness, a sort of long term stamina that limits the character's ability to be extremely active 24/7.
+Report faction camp build sites and improve tutorial.
```

Comment:
```suggestion
Adds weariness, a sort of long-term stamina that limits characters' ability to be extremely active 24/7.
Report faction camp build sites and improve faction camp tutorial.
```
Am I correct re "faction camp" regarding the tutorial? (I've never built one.)

Reply:
I was actually puzzeled by that too, but now that you mention it, that's what it is.